src/worker/core/team/valueChange.ts

```diff
@@ -629,11 +638,15 @@ const getModifiedPickRank = async (
 	pidsAdd: number[],
 	pidsRemove: number[],
 ) => {
+	// later we need to find the new ranks of this team's ovr/estimated win%
+	// it's cleaner to determine this by temporarily removing the old team info from the cached lists
+	const newTeamOvrs = cache.teamOvrs.filter(t => t.tid != tid);
+	// we know this team's old proj win%'s index based on what their old estimated pick was
+	const newWps = cache.wps.filter((wp, i) => i != cache.estPicks[tid] - 1);
```

Comment:
This does work... but since this is the only place `cache.wps` is used, could you change it to be an easier format to work with? Like `{tid: number, wp: number}[]` rather than `number[]`

Reply:
Also can you change the two `!=` to `!==`? I don't like having to think about the difference, so I always use triple equal unless I actually need the weirdness of double equal, which in this case we don't.